velox/docs/configs.rst

```diff
@@ -919,3 +919,66 @@ Tracing
      - integer
      - 0
      - The max trace bytes limit. Tracing is disabled if zero.
+
+Cache
```

Comment:
Let's split `Async Data Cache (In-Memory)`  and `SSD Cache` into 2 different tables.
I also see some configs like `cache.no_retention` in the general config above. Let's move that here.